### ISSUE TITLE
Release of pretrained weights

### ISSUE THREAD

**Issue by chihkuanyeh** [PatrykChrabaszcz/Imagenet32_Scripts#2]
Will the author kindly release pretrained weights for the Imagenet64?

**Comment by PatrykChrabaszcz** [comment]
Hi @yankeesrules 

During the training weights were always overwritten with the weights from the last iteration (to make it possible to recover the training process from the last saved point)
We observe that for big models (k=5) with our regularization after the last learning rate drop model starts to overfit. 
In the report we put results after 31 training epochs for each setting. 

If you are interested, I uploaded weights after 40 epochs of training:
https://drive.google.com/drive/folders/1SuTz4MZqEeU2U21pscQWvy45eTjusiWR?usp=sharing

Hope it will help, in case it is not compatible with the current github code write me back

**Comment by PatrykChrabaszcz** [comment]
Hi @yankeesrules 
Does it work?
